packages/server/database/types/TaskIntegrationJiraServer.ts

```diff
@@ -0,0 +1,16 @@
+import BaseTaskIntegration from './BaseTaskIntegration'
+
+interface Input {
+  accessUserId: string
+  providerId: number
+}
+
+export default class TaskIntegrationJiraServer extends BaseTaskIntegration {
+  providerId: number
+  service!: 'jiraServer'
+  constructor(input: Input) {
+    const {accessUserId, providerId} = input
+    super({accessUserId, service: 'jiraServer'})
+    this.providerId = providerId
```

Comment:
-1 we'll need an issueKey (and maybe projectKey?) in here, too

Reply:
@mattkrick now storing `issueId` and `repositoryId` here too